packages/block-library/src/latest-posts/block.json

```diff
@@ -63,7 +63,7 @@
 		},
 		"featuredImageAlign": {
 			"type": "string",
-			"enum": [ "left", "center", "right" ]
+			"default": "none"
```

Comment:
As you can see from the E2E test failure, updating the block.json definition may affect existing blocks. We should be able to replace it with `ToggleGroupControl` without updating block.json. That is, an `undefined` value is considered to be `'none'`.

Example:

```javascript
<ToggleGroupControl
	label={ __( 'Image alignment' ) }
	value={ featuredImageAlign || 'none' }
	onChange={ ( value ) =>
		setAttributes( {
			featuredImageAlign: value !== 'none' ? value : undefined,
		} )
	}
	isBlock
>

</ToggleGroupControl>
```